EAS energy model source: https://github.com/SiddharthBharadwaj/android_kernel_xiaomi_jasmine_sprout/commit/a5ab1f9d7cb188ef3e20a50c79ee64342d1bb92d
By hypeartist <hypeartist@gmail.com>

Pzqqt: Add a missing special frequency of sdm636 (LITTLE: 1612800)(by linear interpolation).
--- 
+++ 
@@ -17067,29 +17067,29 @@
 		compatible = "sched-energy";
 
 		core-cost0 {
-			busy-cost-data = <0x9ab00 0x29 0xdc500 0x46 0x10fe00 0x53 0x156300 0x92 0x177000 0x9e 0x189c00 0xb7>;
-			idle-cost-data = <0x14 0x10 0x0c 0x08>;
+			busy-cost-data = <0x9ab00 0x443 0xdc500 0x62b 0x10fe00 0x811 0x156300 0xbda 0x177000 0xe66 0x189c00 0xfe9>;
+			idle-cost-data = <0x04 0x04 0x00 0x00>;
 			linux,phandle = <0x05>;
 			phandle = <0x05>;
 		};
 
 		core-cost1 {
-			busy-cost-data = <0x10fe00 0x133 0x156300 0x1e5 0x1aa900 0x359 0x1b8a00 0x373>;
-			idle-cost-data = <0x64 0x50 0x3c 0x28>;
+			busy-cost-data = <0x10fe00 0x134f 0x156300 0x1e6d 0x1aa900 0x35c6 0x1b8a00 0x3b4d>;
+			idle-cost-data = <0x06 0x06 0x00 0x00>;
 			linux,phandle = <0x0e>;
 			phandle = <0x0e>;
 		};
 
 		cluster-cost0 {
-			busy-cost-data = <0x9ab00 0x04 0xdc500 0x05 0x10fe00 0x07 0x156300 0x09 0x177000 0x09 0x189c00 0x0a>;
-			idle-cost-data = <0x04 0x03 0x02 0x01>;
+			busy-cost-data = <0x9ab00 0x16 0xdc500 0x1b 0x10fe00 0x20 0x156300 0x28 0x177000 0x2d 0x189c00 0x30>;
+			idle-cost-data = <0x29 0x29 0x29 0x00>;
 			linux,phandle = <0x06>;
 			phandle = <0x06>;
 		};
 
 		cluster-cost1 {
-			busy-cost-data = <0x10fe00 0x0e 0x156300 0x11 0x1aa900 0x19 0x1b8a00 0x1f>;
-			idle-cost-data = <0x04 0x03 0x02 0x01>;
+			busy-cost-data = <0x10fe00 0x36 0x156300 0x4a 0x1aa900 0x74 0x1b8a00 0x7d>;
+			idle-cost-data = <0x32 0x32 0x32 0x00>;
 			linux,phandle = <0x0f>;
 			phandle = <0x0f>;
 		};
